src/sentry/search/snuba/executors.py

```diff
@@ -465,34 +475,112 @@ def better_priority_aggregation(
     end: datetime,
     aggregate_kwargs: PrioritySortWeights,
 ) -> Sequence[str]:
-    issue_age_weight = 1  # [0, 5]
-    event_age_weight = 1  # [0, 5]
+    issue_age_weight = 1  # [1, 5]
+    event_age_weight = 1  # [1, 5]
     log_level_weight = aggregate_kwargs["log_level"]  # [0, 10]
     stacktrace_weight = aggregate_kwargs["has_stacktrace"]  # [0, 3]
     max_pow = 16
```

Comment:
I wonder if we should increase `max_pow` to a larger number. This is the upper bound of `(event or issue age_hours) / (event or issue halflife hours)`, i.e. any event or issue age that is greater than max_pow times the half-life hours will get clipped.

`2^16 = 65536` isn't very much and will limit the contributions of recent events:
`16 = event_age_hours/event_halflife_hours` -> `16*4 = event_age_hours`. Events barely older than 64 hours (2.66 days) will get pinned to the maximum, same as extremely old events (89 days).

Although, we are taking the inverse of it `1/(2^max_pow) ~= 0.0000152` which is a really small number so maybe its fine?

Reply:
I think the thought there was that once we got 2^16 the result was so small that it didn't matter. We could increase it though if you think it's helpful - mostly I didn't want it getting too close to 32 in case there are any efficiency reasons to not exceed an int.